src/index.js

```diff
@@ -1,19 +1,32 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import path from "path";
 
-import reducers from './reducers/index';
+import reducers from "./reducers/index";
 
-import { createStore, applyMiddleware } from 'redux';
-import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from "redux";
+import { Provider } from "react-redux";
 
-import thunk from 'redux-thunk';
+import thunk from "redux-thunk";
 
-const store = createStore(reducers, applyMiddleware(thunk))
+// Obtain the parent directory for any config
+const directoryAsArr = process.cwd().split("/");
```

Comment:
Is this still needed now that we're bootstrapping the dotenv that react-scripts uses?

Reply:
No it is not and should be removed before being merged.